### PR TITLE
Run bundle exec rubocop -a

### PR DESCRIPTION
**Pull request by agrare**

### CODE REVIEW

**Comment by agrare** [comment]
Some of these come from the generated plugin code, I'll open a PR to fix that as well

File: manageiq-providers-ibm_power_hmc.gemspec [hunk 1]
```diff
@@ -18,7 +19,7 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
-  spec.add_dependency("ibm_power_hmc", [">= 0.1.0"])
+  spec.add_dependency "ibm_power_hmc", "~> 0.1"
```
Comment (agrare) [inline manageiq-providers-ibm_power_hmc.gemspec:22]: The `~> 0.1` wasn't from `rubocop -a` that was me, but we don't want to be doing `>=` or we'll pull in breaking major versions automatically

**Comment by agrare** [comment]
https://github.com/ManageIQ/manageiq/pull/21469

**Review by jaywcarman — APPROVED** [review]
LGTM

**Pull request merged**